Major Update (#268)
NilPointer-Software/bdfd-wiki: src/bdscript/newModal.md

```

> $newModal[Modal ID;Title]

Reviewer: ```suggestion
$newModal[modal ID;title]

```

```
### Parameters

> - `Modal ID` `(Type : String || Flag : Required)` : ID of the `$newModal[]` which will be used for `$onInteraction[]` callback.
> - `Title` `(Type : String || Flag : Required)` : title displayed in the modal.

Reviewer: ```suggestion
- `modal ID` `(Type: String || Flag: Required)`: ID of the `$newModal[]` which is used in `$onInteraction[]` callback.
- `title` `(Type: String || Flag: Required)`: Title displayed in the modal.
```
